proto/osmosis/protorev/v1beta1/genesis.proto

```diff
@@ -25,14 +25,13 @@ message GenesisState {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"base_denoms\""
   ];
-  // The pool weights that are being used to calculate the weight (compute cost)
-  // of each route.
-  //
-  // DEPRECATED: This field is deprecated and will be removed in the next
-  // release. It is replaced by the `info_by_pool_type` field.
+  // DEPRECATED: The pool weights that are being used to calculate the weight
+  // (compute cost) of each route. This field is deprecated and will be removed
+  // in the next release. It is replaced by the `info_by_pool_type` field.
```

Comment:
Drive by change, Skip team agreed this should be marked as deprecated.

Reply:
LGTM, though I'm confused why we don't just mark this as reserved?

Reply:
No you are right we can deprecate, I can make the change now

Reply:
Reserved here https://github.com/osmosis-labs/osmosis/pull/7508/commits/a838fc1f5aa81585f315c47972eed9045e7b877a